migration: add callout for rescheduling mechanism

ryancharris opened the pull request:
## What are you changing in this pull request and why?
recently, the multicell team added a component to the in-app migration guide which allows customers to postpone their account migration by up to 45 days (this functionality is currently only available to enterprise tier accounts).

as we begin migrating more enterprise accounts, we want our public facing docs to highlight this for our users.



<!--
Describe your changes and why you're making them. If related to an open issue or a pull request on dbt Core or another repository, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).
<!--
PRE-RELEASE VERSION OF dbt (if so, uncomment):
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
<!-- 
ADDING OR REMOVING PAGES (if so, uncomment):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages
-->

aaronraff commented on website/docs/docs/cloud/migration.md:
```diff
@@ -15,6 +15,13 @@ Your account will be automatically migrated on or after its scheduled date. Howe
+
+Customers on dbt Cloud Enterprise tiers can postpone their account migration by up to 45 days. To reschedule your migration, go to **Account Settings** → **Migration guide**.
+
+For help, contact us at [sales@getdbt.com](mailto:sales@getdbt.com).
```
Should this be our support email instead, or is sales appropriate for Enterprise accounts?

ryancharris replied:
@aaronraff I believe this should be `support@getdbt.com`.

ryancharris committed:
migration: add callout for rescheduling mechanism

recently, the multicell team added a component to the in-app
migration guide which allows customers to postpone their
account migration by up to 45 days (this functionality is
currently only available to enterprise tier accounts).

as we begin migrating more enterprise accounts, we want our
public facing docs to highlight this for our users.

ryancharris committed:
Merge branch 'current' into MUL-1181-docs-update-migration-guide-documentation-to-reflect-approval-mechanism

matthewshaver commented on website/docs/docs/cloud/migration.md:
```diff
@@ -15,6 +15,13 @@ Your account will be automatically migrated on or after its scheduled date. Howe
 
+:::info Rescheduling your migration
+
+Customers on dbt Cloud Enterprise tiers can postpone their account migration by up to 45 days. To reschedule your migration, go to **Account Settings** → **Migration guide**.
```
Just to keep the language consistent. We generally use 2nd person.
```suggestion
If you're on the dbt Cloud Enterprise tier, you can postpone your account migration by up to 45 days. To reschedule your migration, navigate to **Account Settings** → **Migration guide**.
```

matthewshaver commented on website/docs/docs/cloud/migration.md:
```diff
@@ -15,6 +15,13 @@ Your account will be automatically migrated on or after its scheduled date. Howe
+
+Customers on dbt Cloud Enterprise tiers can postpone their account migration by up to 45 days. To reschedule your migration, go to **Account Settings** → **Migration guide**.
+
+For help, contact us at [support@getdbt.com](mailto:support@getdbt.com).
```
```suggestion
For help, contact the dbt Support Team at [support@getdbt.com](mailto:support@getdbt.com).
```

matthewshaver committed:
Apply suggestions from code review

matthewshaver committed:
Merge branch 'current' into MUL-1181-docs-update-migration-guide-documentation-to-reflect-approval-mechanism
